integreat_cms/cms/models/external_calendars/external_calendar.py

```diff
@@ -0,0 +1,71 @@
+import icalendar
+import requests
+from django.conf import settings
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from integreat_cms.cms.models.abstract_base_model import AbstractBaseModel
+from integreat_cms.cms.models.regions.region import Region
+
+
+class ExternalCalendar(AbstractBaseModel):
+    """
+    Model for representing external calendars, from which events can be imported.
+    """
+
+    name = models.CharField(max_length=255, verbose_name=_("calendar name"), default="")
+    region = models.ForeignKey(
+        Region,
+        on_delete=models.CASCADE,
+        verbose_name=_("region"),
+        related_name="external_calendars",
+    )
+    url = models.URLField(max_length=250, verbose_name=_("URL"))
+
+    import_filter_tag = models.CharField(
+        max_length=255,
+        blank=True,
+        default=settings.EXTERNAL_CALENDAR_TAG,
```

Comment:
The iCalendar specification speaks of categories, not tags. I would like to at least let our internal terminology agree with the official one. The only exception I can think of would be if we already had event categories as a concept in integreat that couldn't overlap with this.
This is the only thing I would rather prefer doing in this PR, as we would need to create a data migration in the next PR otherwise.

```suggestion
    import_filter_category = models.CharField(
        max_length=255,
        blank=True,
        default=settings.EXTERNAL_CALENDAR_CATEGORY,
```